whetstone/compiler/src/main/kotlin/com/freeletics/mad/whetstone/WhetstoneCodeGenerator.kt

```diff
@@ -214,4 +223,25 @@ public class WhetstoneCodeGenerator : CodeGenerator {
     private fun FqName.packageString(): String {
         return pathSegments().joinToString(separator = ".")
     }
+
+    private fun TopLevelFunctionReference.parameterTypes(): List<ComposableParameter> {
+        return parameters
+            .filter { it.name != null && it.name != "state" && it.name != "sendAction" }
```

Comment:
`state` and `sendAction` are specific to our codebase. Not sure how this could be prevented

Reply:
I think this is fine, for a first step. A better long term solution would be to see how we can get the type parameters of the state machine class and then match based on those. E.g. FooStateMachine which we have extends `StateMachine<FooState, FooAction>` so we can extract those 2 to use them here.